src/main/java/core/basesyntax/CustomFileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CustomFileReaderImpl implements CustomFileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath))) {
+            StringBuilder builder = new StringBuilder();
+            String line = reader.readLine();
+            while (line != null) {
+                lines.add(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + filePath, e);
```

Comment:
The exception message should include a space before the file path for better readability. It should be "Can't read file " + filePath.